### PR TITLE
Refactor of PathVerificationService

### PR DESCRIPTION
**Pull request by niksv**
Part of #580

---
This change is [Reviewable](https://reviewable.io/reviews/telstra/open-kilda/2316)

### CODE REVIEW

File: services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/pathverification/PathVerificationService.java [hunk 1]
```diff
@@ -366,22 +369,23 @@ private LLDPTLV makeIdLldptvPacket(byte[] data, byte type) {
         return new LLDPTLV().setType(type).setLength((short) data.length).setValue(data);
     }
 
-    private VerificationPacket deserialize(Ethernet eth) throws Exception {
+    VerificationPacket deserialize(Ethernet eth) {
```
Comment (dpoltavets) [inline services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/pathverification/PathVerificationService.java]: minor: Why did you remove private access modifier?

Comment (niksv) [reply services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/pathverification/PathVerificationService.java]: for testing. Will add annotation
